lib/02_calculator.rb

```diff
@@ -0,0 +1,41 @@
+def add(a,b)
+  a.to_f + b.to_f
+end
+
+def subtract(a,b)
+  a.to_f - b.to_f
+end
+
+def sum(nums)
+  total = 0
```

Comment:
You can accomplish the same by replacing all lines in this method with `nums.reduce(:+)`. (Though, this method works perfectly well too.)